models/kyberswap/polygon/kyberswap_polygon_schema.yml

```diff
@@ -0,0 +1,142 @@
+      - &token_bought_symbol
+        name: token_bought_symbol
+        description: "Token symbol for token bought in the transaction"
+      - &token_sold_symbol
+        name: token_sold_symbol
+        description: "Token symbol for token sold in the transaction"
+      - &token_pair
+        name: token_pair
+        description: "Token symbol pair for each token involved in the transaction"
+      - &token_bought_amount
+        name: token_bought_amount
+        description: "Value of the token bought at time of execution in the original currency"
+      - &token_sold_amount
+        name: token_sold_amount
+        description: "Value of the token sold at time of execution in the original currency"
+      - &token_bought_amount_raw
+        name: token_bought_amount_raw
+        description: "Raw value of the token bought at time of execution in the original currency"
+      - &token_sold_amount_raw
+        name: token_sold_amount_raw
+        description: "Raw value of the token sold at time of execution in the original currency"
+      - &amount_usd
+        name: amount_usd
+        description: "USD value of the trade at time of execution"
+      - &token_bought_address
+        name: token_bought_address
+        description: "Contract address of the token bought"
+      - &token_sold_address
+        name: token_sold_address
+        description: "Contract address of the token sold"
+      - &taker
+        name: taker
+        description: "Address of trader who purchased a token"
+      - &maker
+        name: maker
+        description: "Address of trader who sold a token"
+      - &project_contract_address
+        name: project_contract_address
+        description: "Project contract address which executed the trade on the blockchain"
+      - &tx_hash
+        name: tx_hash
+        description: "Unique transaction hash value tied to each transaction on the DEX"
+      - &tx_from
+        name: tx_from
+        description: "Address which initiated the transaction"
+      - &tx_to
+        name: tx_to
+        description: "Address which received the transaction"
+      - &evt_index
+        name: evt_index
+        description: "the position of this event log within this transaction"
+  - name: kyberswap_aggregator_polygon_trades
+    meta:
+      blockchain: polygon
+      sector: dex_aggregator
+      project: kyberswap
+      contributors: nhd98z
+    config:
+      tags: ['polygon', 'kyberswap', 'aggregator', 'trades', 'dex']
+    description: >
+      Kyberswap Aggregator on polygon.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+            - trace_address
+      - check_dex_aggregator_seed:
+          blockchain: polygon
+          project: kyberswap
+          version: meta_2
```

Comment:
Also for `meta_2` version.
Without the seed, the `check_dex_aggregator_seed` will not work.

Reply:
I did have in `https://github.com/duneanalytics/spellbook/blob/7e9111697e045aad0e7cf32b2fad5a8f172c412c/seeds/dex/aggregator/dex_aggregator_seed.csv#L116`